airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -20,33 +20,52 @@
 """
 This module contains Google BigQuery operators.
 """
+import datetime
 import enum
 import hashlib
 import json
+import logging
+import random
 import re
+import string
 import uuid
 import warnings
 from typing import Any, Dict, Iterable, List, Optional, Sequence, Set, SupportsAbs, Union
+from urllib.parse import urlsplit
 
 import attr
 from google.api_core.exceptions import Conflict
 from google.cloud.bigquery import TableReference
 
+from great_expectations.data_context.types.base import DataContextConfig
```

Comment:
The dependency will need to get added as an extra to setup.py

Reply:
I'm wondering how we should approach it. GE should not be a requirement for other Google operators, on the other hand having separate package for GE doesn't sounds like a good way. Unless we make this operator generic and provider neutral